### ISSUE TITLE
Receiving `ModuleNotFoundError("No module named 'sklearn'")`

### ISSUE THREAD

**Issue by davidshq** [OpenImaging/miqa#638]
When running MIQA I'm now seeing this in the terminal:
```
miqa-oi-celery-1    | [2022-12-05 21:43:19,336: ERROR/ForkPoolWorker-2] Task miqa.core.tasks.evaluate_frame_content[362b83d4-5309-4195-8a19-91fa2f7832f8] raised unexpected: ModuleNotFoundError("No module named 'sklearn'")
miqa-oi-celery-1    | Traceback (most recent call last):
miqa-oi-celery-1    |   File "/usr/local/lib/python3.8/site-packages/celery/app/trace.py", line 451, in trace_task
miqa-oi-celery-1    |     R = retval = fun(*args, **kwargs)
miqa-oi-celery-1    |   File "/usr/local/lib/python3.8/site-packages/celery/app/trace.py", line 734, in __protected_call__
miqa-oi-celery-1    |     return self.run(*args, **kwargs)
miqa-oi-celery-1    |   File "/opt/django-project/miqa/core/tasks.py", line 64, in evaluate_frame_content
miqa-oi-celery-1    |     from miqa.learning.evaluation_models import available_evaluation_models
miqa-oi-celery-1    |   File "/opt/django-project/miqa/learning/evaluation_models.py", line 7, in <module>
miqa-oi-celery-1    |     from miqa.learning.nn_inference import get_model
miqa-oi-celery-1    |   File "/opt/django-project/miqa/learning/nn_inference.py", line 7, in <module>
miqa-oi-celery-1    |     from sklearn.metrics import classification_report, confusion_matrix, mean_squared_error, r2_score
miqa-oi-celery-1    | ModuleNotFoundError: No module named 'sklearn'
```

And the auto evaluations no longer appear to be occurring. At first I thought this had something to do with the changes I've made in my [fork](https://github.com/davidshq/miqa) but I've downloaded a fresh copy of MIQA from this repo and run through the normal install process and am still seeing this error.

I suspect this is related to the [deprecation of the sklearn package](https://pypi.org/project/sklearn/):
> Brownout schedule
> ...
> 2022 December 1st - 2023 January 31st | :00-:05 every hour

If I run `pip list` in the django container I see that `sklearn` (0.0.post1) is installed but scikit-learn isn't.

I'm continuing to work on this but wanted to see if this is an issue everyone is experiencing. My guess is that it will depend on when the instance was spun up. If it has been post Dec 1st then I'd expect the same issue to arise.

Note: `scikit-learn` is required in `learning/requirements.txt` but `sklearn` is required in `setup.py`

**Comment by davidshq** [comment]
Removing `sklearn` using `pip uninstall sklearn` and then installing `scikit-learn` using `pip install scikit-learn` on the django instance didn't make a difference.

However, removing `sklean` and installing `scikit-learn` on celery did.

**Comment by annehaley** [comment]
Resolved by #643
